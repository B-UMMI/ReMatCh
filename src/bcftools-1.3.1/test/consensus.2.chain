chain 500 1 501 + 1 501 1 504 + 1 504 1
15 0 3
485

chain 491 2 501 + 0 501 2 491 + 0 491 2
200 10 0
291

