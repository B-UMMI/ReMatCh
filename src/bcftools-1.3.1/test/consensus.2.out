>1:2-501
TACCATATGTGACATAAAATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTT
TGCAGAAAAGGAAGACTTAAAAAGAGTCAGTACTAACCTACATAATATATACAATGTTCA
TTAAATAATAAAATGAGCTCATCATACTTAGGTCATCATAAATATATCTGAAATTCACAA
ATATTGATCAAATGGTAAAATAGACAAGTAGATTTTAATAGGTTAAACAATTACTGATTC
TCTTGAAAGAATAAATTTAATATGAGACCTATTTCATTATAATGAACTCACAAATTAGAA
ACTTCACACTGGGGGCTGGAGAGATGGCTCAGTAGTTAAGAACACTGACTGCTCTTCTGA
AGGTCCTGAGTTCAAATCCCAGCAACCACATGGTGACTTACAACCATCTGTAATGACATC
TGNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
NNTTTAAAAACAAAAAAAAAGAA
>2
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
AGAGATTAGTGATTTTCCATATTCTTTAAGTCATTTTAGAGTAATGTGTTCTTAAGATAA
ATCAGAAAAACAAAAACTTGTGCTTTCCTGTTTGAAAAACAAACAGCTGTGGGGAATGGT
GTCGGGACAGCCTTTTTATAAAATAATGTTGAGGCTTTGATACGTCAAAGNNNNNNNNNN
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNTTTGCTGCTG
CCAATGACAGCACACCCTGGGAATGCCCCAACTACTTACTACAAAGCAGTGTTACATGGA
GAAGATCTTCAAGAGTCTTTTTGCTAGATCTTTCCTTGGCTTTTGATGTGACTCCTCTCA
ATAAAATCCACAGTAATATAGTGAGTGGTCTCCTGCTCCAAACCAGTATTTCAGACACAG
TTAATCCAGAC
